_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: default
+title: Star-tree
+nav_order: 61
+parent: Supported field types
+---
+
+# Star-tree field type
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+A [star-tree index](https://docs.pinot.apache.org/basics/indexing/star-tree-index) precomputes aggregations, accelerating the performance of aggregation queries. 
+If a star-tree index is configured as part of an index mapping, the star-tree index is created and maintained as data is ingested in real time.
+
+OpenSearch will automatically use the star-tree index to optimize aggregations if the queried fields are part of star-tree index dimension fields and the aggregations are on star-tree index metric fields. No changes are required in the query syntax or the request parameters.
+
+For more information, see [Star-tree index]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index/).
+
+## Prerequisites
+
+To use a star-tree index, follow the instructions in [Enabling a star-tree index]({{site.url}}{{site.baseurl}}/search-plugins/star-tree-index#enabling-the-star-tree-index).
+
+## Limitations
+
+The star-tree index feature has the following limitations:
+
+- A star-tree index should only be used on indexes whose data is not updated or deleted because standard updates and deletions are not accounted for in a star-tree index.
```

Comment:
```suggestion
- A star-tree index should only be enabled on indexes whose data is not updated or deleted because standard updates and deletions are not accounted for in a star-tree index.
```